libkbfs/block_types.go

```diff
@@ -188,10 +188,30 @@ func (fb *FileBlock) NewEmpty() Block {
 
 // DataVersion returns data version for this block.
 func (fb *FileBlock) DataVersion() DataVer {
+	// If this is an indirect block, and none of its children are
+	// marked as direct blocks, then this must be a big file.  Note
+	// that we do it this way, rather than returning on the first
+	// non-direct block, to support appending to existing files and
+	// making them big.
+	hasHoles := false
```

Comment:
When the holes code was written the various operations operated on (Indirect)FilePtrs rather than FileBlocks iirc, but after some libkbfs refactorings if it would be written *now* having it in FileBlocks would be probably better.